### PR TITLE
Add top-level checkbox to turn markers on/off for all plot data series

### PR DESCRIPTION
**Pull request by kd0aij**
could be expanded to add a symbol selection field for each data series

### CODE REVIEW

**Comment by AndreasAntener** [comment]
Looks good. I think it would also be good if we'd add the marker state to the preferences. Have a look at save/loadPreferences methods. You could just put it in the top-level preference instance.

File: src/me/drton/flightplot/FlightPlot.java [hunk 1]
```diff
@@ -239,6 +242,20 @@ public void actionPerformed(ActionEvent e) {
             }
         });
         mainFrame.setVisible(true);
+        markerCheckBox.addComponentListener(new ComponentAdapter() {
```
Comment (AndreasAntener) [inline src/me/drton/flightplot/FlightPlot.java]: is this needed?

**Comment by kd0aij** [comment]
the checkbox listener causes markers to appear/disappear when you check/uncheck the box.

**Comment by AndreasAntener** [comment]
Yes but I'm referring to the line above the action listener.

**Comment by kd0aij** [comment]
OH. Is that garbage code coming from the UI editor? Seems like it can't do anything useful.

**Comment by AndreasAntener** [comment]
Quite possible ;), don't know IntelliJ well enough..

**Comment by AndreasAntener** [comment]
@kd0aij I copied your commits to a new branch with some fixes (#43), closing this

**Pull request closed without merging**